pylintrc

```diff
@@ -207,7 +206,8 @@ enable=c-extension-no-member,
        singleton-comparison,
        use-dict-literal,
        use-list-literal,
-       unnecessary-lambda-assignment
+       unnecessary-lambda-assignment,
+       superfluous-parens
```

Comment:
we don't need to enable this. Only remove it from the `disable` list as you did on line 150.